src/hook/manager.cpp

```diff
@@ -47,16 +47,16 @@ using mesos::modules::ModuleManager;
 namespace mesos {
 namespace internal {
 
-static std::mutex mutex;
-static LinkedHashMap<string, Hook*> availableHooks;
+static std::mutex* mutex = new(std::mutex);
+static LinkedHashMap<string, Hook*>* availableHooks = new(LinkedHashMap<string, Hook*>);
```

Comment:
I'm not familiar with this `new` syntax, we would use:

```
static std::mutex* mutex = new std::mutex();
static LinkedHashMap<string, Hook*>* availableHooks = new LinkedHashMap<string, Hook*>();
```

Reply:
This line is also > 80 characters. Are you using the commit hook? It would catch this issue:

http://mesos.apache.org/documentation/latest/advanced-contribution/#create-your-patch

Reply:
Thanks for your info, just installed the commit hook and make changes to the code accordingly per your suggestion.